### PR TITLE
Add utilities to delete a tag and assign a license to an org's datasets

### PR DESCRIPTION
**Pull request by sharph**

### CODE REVIEW

File: odp_importer/assignlicense4org.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        if license['id'] == license_id or license['title'] == license_id:
+            return license
+    return None
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Set a license for all '
+                                                 'datasets owned by an '
+                                                 'organization')
+    parser.add_argument('org', metavar='ORG', help='Organization')
```
Comment (ddohler) [inline odp_importer/assignlicense4org.py]: It'd be good to expand the help text to be clear that this is the `name` field (I think that's the CKAN term for the slug) for the organization. Same for the licenses.

**Comment by ddohler** [comment]
+1

**Comment by sharph** [comment]
I added some code to allow setting the license for many orgs based on the prefix. ("City of Phila -")

**Comment by ddohler** [comment]
Looks good to me!

**Pull request merged**